README.md

```diff
@@ -1,168 +1,156 @@
-# Alternate Loading Modes
+# Prerendering, revamped
```

Comment:
One option is to use a more general term like "speculation" instead, that captures dns-prefetch, preconnect, prefetch, and prerender. Obviously some pieces are prerender-specific. But obviously the more general a term we use the less clear it will be at first glance what we mean.

Reply:
I also think the ladder of speculation is interesting. If there are too many links that would be okay to prerender and we're not sure, I think we should be free to prefetch the 3 most likely options instead, for instance.

Reply:
My take was that we should focus this around prerendering, and say that we might capture some value for prefetching etc. along the way. But it's easier to paint a coherent picture as to why these all belong as part of the same effort, if that effort is prerendering.

Reply:
I've pushed a commit that expands the "Prefetching" section to explain this a bit more.